ember-qunit-codemod.js

```diff
@@ -419,39 +416,43 @@ module.exports = function(file, api, options) {
     let programPath = root.get('program');
     let bodyPath = programPath.get('body');
 
-    let bodyReplacement = [];
     let currentModuleCallbackBody, currentTestType, currentSubject, currentHasCustomSubject;
     bodyPath.each(expressionPath => {
       let expression = expressionPath.node;
       if (isModuleDefinition(expressionPath)) {
         let result = createModule(expressionPath);
-        bodyReplacement.push(result[0]);
+        expressionPath.replace(result[0]);
         currentModuleCallbackBody = result[1];
         currentTestType = result[2];
         currentSubject = result[3];
         currentHasCustomSubject = result[4];
       } else if (currentModuleCallbackBody) {
+        // calling `path.replace()` essentially just removes
```

Comment:
did you finish your thought?

Reply:
Hehe, yeah but it does seem to trail off. Basically I’m saying it’s like `path.node.remove()` but only effects the specific location (so we can still reinsert it into the AST in the new nested location).